python/tests/tests.py

```diff
@@ -98,9 +111,13 @@ def test_template(self):
         exp = [
             {"name": "fred", "age": 20},
             {"name": "pete", "age": 30},
-            {"age": 25}
+            {"name": None, "age": 25}
```

Comment:
this is the only behavioral change; previously we would not set `name` if we were SKIPing a value.  Nothing generated this in practice and it makes things more regular if we return None instead